.babelrc

```diff
@@ -0,0 +1,27 @@
+{
+  "presets": [ [ "env" ] ],
+  "plugins": [
+    [
+      "transform-react-jsx",
+      {
+        "pragma": "wp.element.createElement"
+      }
+    ],
+    "dynamic-import-webpack",
+    "transform-object-rest-spread"
+  ],
+  "env": {
+    "test": {
+      "plugins": [
+        [
+          "transform-react-jsx",
+          {
+            "pragma": "wp.element.createElement"
+          }
+        ],
+        "babel-plugin-styled-components"
```

Comment:
To my understanding, this whole .babelrc file is copy of the one used on Yoast SEO which is totally fine I guess. Only thing: I'm not sure we're going to use styled-components.